about.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="about.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>about page</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+
+    <header>
+    <div class="above-nav">
+            <a href="www.facebook.com"><i class="fa-brands nav-icon fa-facebook-f"></i></a>
+            <a href="www.twitter.com"><i class="fa-brands nav-icon fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="about.html">About</a>
+    </div>        
+        <nav class="nav">
+            <a href="index.html"><img id="nav-logo" src="images/base/global-logo.png" alt="home page logo"></a>
+            <ul class="nav-list">
+                <li><a id="about" href="index.html">Home</a></li>
+                <li><a href="#">program</a></li>
+                <li><a href="#">join</a></li>
+                <li><a href="#">news</a></li>
+                <li><a id="nav-button" href="#">CC campaign</a></li>
+            </ul>
+                <img class="close" src="images/base/close_mobile_menu.png" alt="" srcset="">
+                <img class="burger" src="images/base/mobile_menu.png" alt="burger icon" srcset="">
+        </nav>
+        </header>
+        <section class="hero">
+                <div class="hero-cont">
+                    <h1>Hello sharing world <br> 11th global<br> conference on global <br> <span id="span-head"> warming </span></h1>
+                    <p>The 11th Global Conference on Global Warming (GCGW-2023) is a multi disciplinary international conference on all aspects.The 11th Global Conference on Global Warming (GCGW-2023) is a multi disciplinary international conference on all aspects.</p>
+                    <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+                </div>
```

Comment:
- [ OPTIONAL ] Greay job working on the required changes but I have a small note to add here in the paragraph you should make the color for the text darker as shown in the given design since in your design is gray but it should be darker so you can use this color ⬇ to make it mre clear to the eye  as well for accessiblity reasons💪🏻🤩 and I am assiging as optional because the previous reviewer didn't mention it but it still important to make 💪🏻


![image](https://github.com/MohamedRoshdy2021/Conference-project/assets/58553711/11ff4426-f000-41ee-948f-c13917f70c47)


- Your
![image](https://github.com/MohamedRoshdy2021/Conference-project/assets/58553711/d6dbb640-fb8e-4d4f-af4b-176519f1368b)


- given
![image](https://github.com/MohamedRoshdy2021/Conference-project/assets/58553711/f4a9e70e-c1d4-423e-98a1-a2febd6015d8)